jenkins-schema.sh

```diff
@@ -38,6 +38,7 @@ git checkout $SCHEMA_GIT_COMMIT
 cd ../..
 
 time bundle install --path "${HOME}/bundles/${JOB_NAME}" --deployment
+bundle exec rake db:reset
```

Comment:
That sets the env globally though: https://github.com/alphagov/policy-publisher/blob/master/jenkins.sh#L3